CMakeLists.txt

```diff
@@ -566,9 +566,45 @@ if("${DREAM3D_DATA_DIR}" STREQUAL "")
   endif()
 endif()
 
+# -----------------------------------------------------------------------------
+# Here we are going to setup to download and decompress the test files. In order
+# to add your own test files you will need to tar.gz the test file, compute the
+# SHA 512 Hash of the file and then upload the file to 
+# https://github.com/BlueQuartzSoftware/complex/releases/tag/Data_Archive.
+#
+# Go to the web site above, "edit" the release, add the filename and SHA 512 to
+# the table and then upload your compressed file.
+# Save the release so that the repo is updated
+# -----------------------------------------------------------------------------
```

Comment:
@imikejackson This would be useful information to add to the PortingFilter markdown file distributed in complex_plugins and complex. Maybe we should extent a section in that file for unit testing or even set up a second markdown file specifically for outlining unit testing nuances. Especially since we seem to be having the most trouble getting rigorous testing implemented with the porting.

Reply:
@nyoungbq That is a good idea.